### ISSUE TITLE
at-rule-blockless-newline-after

### ISSUE THREAD

**Issue by evilebottnawi** [stylelint/stylelint#897]
> Describe the issue. Is it a bug or a feature request (new rule, new option, etc.)?

Disallow multiple at-rule `import` in one line.

> Which rule, if any, is this issue related to?

New rule `at-rule-one-import-declaration-per-line`

> What CSS is needed to reproduce this issue?

```css
@import url("x.css"); @import url("y.css");
@import url("x.css");
```

> What stylelint configuration is needed to reproduce this issue?

```json
{
  "rules": {
    "at-rule-one-import-declaration-per-line": 2
  }
}
```

> Which version of stylelint are you using?

`4.5.1`

> How are you running stylelint: CLI, PostCSS plugin, Node API?

PostCSS plugin

> Does your issue relate to non-standard syntax (e.g. SCSS, nesting, etc.)?

SCSS

> What did you expect to happen?

Throw error if `import` declaration not one in same line.

> What actually happened (e.g. what warnings or errors you are getting)?

Nothing

**Comment by jeddy3** [comment]
It's been a while since we added an `*-after` rule, but this seems like a valid request :)

`at-rule-blockless-newline-after: "always"`? 

This rule will check more than just `@import`s e.g. `@charset` and `@namespace` etc...

I don't think we need to add any other options or companion rules (e.g. `space-after`) until it's requested.

**Comment by evilebottnawi** [comment]
@jeddy3 @davidtheclark rule created, close this issue?

**Comment by ntwb** [comment]
Yes @evilebottnawi, closing this per:
• Added `at-rule-blockless-newline-after` in #932 
• Renamed rule to `at-rule-semicolon-newline-after` in #954/#956
